include/fizzy/fizzy.h

```diff
@@ -75,6 +97,13 @@ const FizzyModule* fizzy_parse(const uint8_t* wasm_binary, size_t wasm_binary_si
 /// If passed pointer is NULL, has no effect.
 void fizzy_free_module(const FizzyModule* module);
 
+/// Get type of the function defined in the module.
+///
+/// @param module   Pointer to module.
+/// @param func_idx Function index. Can be index of an imported function.
```

Comment:
Should mention how the indexes are following eachother (imports vs. module defs).